src/go/cmd/update-collection-v3/migrations/metrics/valuesv2.go

```diff
@@ -0,0 +1,14 @@
+package metrics
+
+type ValuesV2 struct {
```

Comment:
It's misleading to name it "v2" because this is not really the "v2" version of values - it's something in between v2 and v3.